src/common/functions.rs

```diff
@@ -580,25 +580,25 @@ mod format_duration_tests {
     #[test]
     fn test_one_minute_ago() {
         let past_time = get_past_time(Duration::minutes(1));
-        assert_eq!(print_time_since(&past_time.to_rfc3339()), "1 minute ago");
+        assert_eq!(print_time_since(&past_time.to_rfc3339()), "1 min ago");
     }
 
     #[test]
     fn test_multiple_minutes_ago() {
         let past_time = get_past_time(Duration::minutes(20));
-        assert_eq!(print_time_since(&past_time.to_rfc3339()), "20 minutes ago");
+        assert_eq!(print_time_since(&past_time.to_rfc3339()), "20 mins ago");
```

Comment:
If you were to standardize on the standard (SI), then there would be no requirement to pluralize.